datasetinsights/datasets/simulation/__init__.py

```diff
@@ -0,0 +1,26 @@
+from .captures import Captures
```

Comment:
Since we are only changing folder structure in this review, maybe this change should be pushed for the next review where you are reorganizing modules?

Reply:
Its binding in init at same level. Need to import to fix the dependency.

Reply:
If this review is focused on folder structure change, these import "shortcuts" are our of the scope for this review?